### ISSUE TITLE
Scrolling for Help and Accessibility statement pages

### ISSUE THREAD

**Issue by agbeltran** [ral-facilities/scigateway#996]
**Description**:
Scrolling in browse/search works so that the top of the page, menu and footer are always visible.

We should have the Help and Accessibility statement have the same behaviour.

**Acceptance criteria**:
- [ ] Help and Accessibility statement support scrolling by keeping top of the page, menu and footer.
